src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderMovementApplication.kt

```diff
@@ -0,0 +1,179 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.OneToOne
+import jakarta.persistence.SequenceGenerator
+import jakarta.persistence.Table
+import org.hibernate.annotations.JoinColumnOrFormula
+import org.hibernate.annotations.JoinColumnsOrFormulas
+import org.hibernate.annotations.JoinFormula
+import org.hibernate.annotations.NotFound
+import org.hibernate.annotations.NotFoundAction
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "OFFENDER_MOVEMENT_APPS")
+class OffenderMovementApplication(
```

Comment:
The main change is to model this table which is a mandatory application for a temporary absence.